common/changes/@microsoft/rush/phased-commands_2021-12-25-04-47.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "(BREAKING CHANGE) Remove the \"write-build-cache\" action.",
```

Comment:
```suggestion
      "comment": "(BREAKING CHANGE) Remove the experimental command \"rush write-build-cache\", since it is no longer needed and would be incompatible with phased builds. If you need this command for some reason, please create a GitHub issue.",
```